.github/workflows/performance.yml

```diff
@@ -32,17 +32,19 @@ permissions: {}
 jobs:
   # Runs the performance test suite.
   performance:
-    name: Performance tests ${{ matrix.memcached && '(with memcached)' || '' }}
-    uses: WordPress/wordpress-develop/.github/workflows/reusable-performance.yml@trunk
+    name: ${{ matrix.multisite && 'Multisite' || 'Single' }}${{ matrix.memcached && ' (memcached)' || '' }}
```

Comment:
Why not use the same name as below, which includes "Performance tests"?
```suggestion
    name: Performance tests / ${{ matrix.multisite && 'Multisite' || 'Single' }}${{ matrix.memcached && ' (memcached)' || '' }}
```

Reply:
This would just be repetitive. I have been experimenting with the workflow names to ensure that we don't have super long and repeating names (everything would be printed twice because the two workflows have the same names). Plus, GitHub can do nice nesting of the jobs if done right.

I was hoping to get @desrosj input on that part.

<img width="326" alt="Screenshot 2024-12-19 at 11 09 24" src="https://github.com/user-attachments/assets/11f754c2-09d5-439c-b7af-07f7215b588e" />
<img width="832" alt="Screenshot 2024-12-19 at 11 09 04" src="https://github.com/user-attachments/assets/21a5c3ae-e760-4943-a196-07a4a92bea26" />

Reply:
This is what I wanted:


<img width="319" alt="Screenshot 2024-12-19 at 11 38 32" src="https://github.com/user-attachments/assets/4d7865c4-9de8-453e-aeed-19f0a1858b62" />

At the cost of repetitive labels everywhere else 🙃

Reply:
Ew, those job summary titles are not great.

Have you tried eliminating the "Performance tests" part of the job title? Then it would be "Single site" and "Multisite" as the top level, and they would be grouped by site type.